packages/now-client/src/utils/index.ts

```diff
@@ -74,15 +75,17 @@ const maybeRead = async function<T>(path: string, default_: T) {
   }
 };
 
-export async function getNowIgnore(path: string | string[]): Promise<any> {
-  let ignores: string[] = [
-    '.hg',
-    '.git',
+export async function getNowIgnore(
+  path: string | string[]
+): Promise<{ ig: Ignore; ignores: string[] }> {
+  const ignores: string[] = [
+    '.hg/',
+    '.git/',
     '.gitmodules',
-    '.svn',
+    '.svn/',
     '.cache',
-    '.next',
-    '.now',
+    '.next/',
+    '.now/',
```

Comment:
This is a workaround for `@now/static-build` which globs the files. Since node-glob ignore option only works with glob syntax and not `.gitignore` syntax, I needed a way to determine which of these are directories. The assumption here is the ones with trailing slash are directories so that `@now/static-build` could change `.git/` into `.git/**` and correctly ignore all the files.